sbin/Makefile.am

```diff
@@ -55,10 +52,65 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
+
+if FILENAME_VERSIONS
+
+install-exec-hook:
+if ENABLE_FIREHOL
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f firehol firehol-@PACKAGE_VERSION@ && \
+		$(LN_S) firehol-@PACKAGE_VERSION@ firehol
+endif
+if ENABLE_FIREQOS
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f fireqos fireqos-@PACKAGE_VERSION@ && \
+		$(LN_S) fireqos-@PACKAGE_VERSION@ fireqos
+endif
+if ENABLE_LINK_BALANCER
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f link-balancer link-balancer-@PACKAGE_VERSION@ && \
+		$(LN_S) link-balancer-@PACKAGE_VERSION@ link-balancer
+endif
+if ENABLE_UPDATE_IPSETS
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f update-ipsets update-ipsets-@PACKAGE_VERSION@ && \
+		$(LN_S) update-ipsets-@PACKAGE_VERSION@ update-ipsets
+endif
+if ENABLE_VNETBUILD
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f vnetbuild vnetbuild-@PACKAGE_VERSION@ && \
+		$(LN_S) vnetbuild-@PACKAGE_VERSION@ vnetbuild
+endif
```

Comment:
I just figured out you meant to write `readlink` so yes, that would work to find the location regardless. I retract the small script as a requirement.

It does leave open the question of how the location of `readlink` is specified if we are not processing the whole script though, in order to be able to read the firehol.config

Or have I misunderstood? i.e. will this satisfy everyone:

1. scripts `firehol.in` etc. are processed by configure to expand paths, then installed to /usr/libexex/whatever-versioned/scriptname

2. links to /sbin made for these

3. firehol.config is introduced alongside the scripts. It holds variables for the paths for the configured directories such as /etc /var etc. and is read based on the location of the script using `readlink` as already proposed.

Maybe that is exactly what you are proposing anyway @alonbl ? If so, do you also agree @ktsaou ? I will get it implemented that way.